source/nodejs/adaptivecards-site/themes/adaptivecards/scripts/generator-adaptiveassets.js

```diff
@@ -2,169 +2,203 @@
 // Licensed under the MIT License.
 "use strict";
 
+const emitDebugOutput = false; // set to true to get debug spew firehose
+
+function debugLog(msg) {
+    if (emitDebugOutput) {
+        console.log(msg);
+    }
+}
+
 // Copy various adaptive card assets from elsehwere in the repo into the output folder
 
 var fs = require("hexo-fs");
 var path = require("path");
 var glob = require("glob");
 var md5 = require("md5");
 
-// These are the flat asset files that will be copied into the output folder
-// and available to reference in HTML templates
+// These are the flat asset files that will be copied into the output folder and available to
+// reference in HTML templates. Note that these are expected to exist within the
+// `adaptivecards-site/node_modules` directory
 var simpleAssets = [
-  "node_modules/adaptivecards/dist/*.*",
-  "node_modules/adaptive-expressions/lib/*.*",
-  "node_modules/adaptivecards-designer/dist/*.*",
-  "node_modules/adaptivecards-templating/dist/*.*",
-  "node_modules/@fortawesome/fontawesome-free/css/all.min.css",
-  "node_modules/@fortawesome/fontawesome-free/webfonts/*.*",
-  "node_modules/highlightjs/highlight.pack.min.js",
-  "node_modules/highlightjs/styles/default.css",
-  "node_modules/jquery/dist/jquery.min.js",
-  "node_modules/markdown-it/dist/markdown-it.min.js"
+    "node_modules/adaptive-expressions/lib/*.*",
```

Comment:
Were some of these unused?

Reply:
they weren't unused, but part of the switch to `npm workspaces` shuffled around where some of the node packages were placed on disk (e.g. instead of `source/nodejs/adaptivecards-site/node_modules/adaptivecards/...` the package is at `source/nodejs/node_modules/adaptivecards/...`). these types of changes are _usually_ transparent to tooling (node walks up parent directories to the project root when trying to resolve packages), but since we have our own hand-rolled stuff for the site, I needed to make these logic changes -- if you look a bit further down, you'll see that we are still including these :)

https://github.com/microsoft/AdaptiveCards/blob/02c2427da2e99f9d60f7d899fddd263888d7ea45/source/nodejs/adaptivecards-site/themes/adaptivecards/scripts/generator-adaptiveassets.js#L36-L45